packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -63,31 +251,66 @@ const BlocksContent = ({ placeholder }: BlocksInputProps) => {
     [modifiers]
   );
 
+  const handleMoveBlocks = (editor: Editor, event: React.KeyboardEvent<HTMLElement>) => {
+    if (editor.selection) {
```

Comment:
since there is no "else", I would recommend to do an early return instead if selection is undefined, so that the code is flatter